basics/file_os_create.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	f, err := os.Create("a.txt")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	l, err := f.WriteString("Hello World")
```

Comment:
Not sure, but is it ok to redeclare the same variable?

Reply:
fixed it in the new commit.